vj4/db.py

```diff
@@ -22,7 +23,8 @@ def escape(s):
     uri = 'mongodb://{}:{}@{}/?authSource={}'.format(escape(options.db_username),
                                                      escape(options.db_password),
                                                      escape(options.db_host),
-                                                     escape(options.db_name))
+                                                     escape(options.db_auth if options.db_auth
```

Comment:
Consider only append `?authSource` when `--db-auth-source` is set, otherwise use `mongodb://u:p@h/name`

Reply:
fyi the code to parse this URI string is here:
https://github.com/ZeoAlliance/aiomongo/blob/adeec47b2ef7e2550a797b3202f6604a9da84d67/aiomongo/client.py#L21